cosmos/stride.json

```diff
@@ -49,7 +49,11 @@
       "coinDenom": "stSTARS",
       "coinMinimalDenom": "stustars",
       "coinDecimals": 6
-    }
+    },
+      "coinDenom": "stLUNA",
+      "coinMinimalDenom": "stuluna",
+      "coinDecimals": 6
+    },
```

Comment:
Could you remove comma(,) line 56?
It doesn't fit the json format.

Reply:
Removed! How does this look?